cmd/resource/status/cmd/util_test.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+	"testing"
```

Comment:
Nit: We usually separate imports from the standard go libraries and third-party libraries.